File: android/app/src/main/java/jaclib/peer/IUnknown.java
package jaclib.peer;


public class IUnknown extends Peer {

    public IUnknown(ti arg0) {
        this.reference = new IUnknownReference(this, arg0);
    }

    public long AddRef() { return 0L; }

    public final long a(int arg0) {
        return super.a();
    }
}

File: src/jvmMain/kotlin/jaclib/peer/IUnknown.kt
package jaclib.peer


open actual class IUnknown actual constructor(arg0: hb?) : Peer() {
    init {
        this.reference = IUnknownReference(this, arg0)
    }

    /*private*/ actual external fun AddRef(): Long

    actual fun a(arg0: Int): Long {
        return super.a()
    }
}
